Core/Utilities/DEncode.py

```diff
@@ -16,236 +16,369 @@
 
 import types
 import datetime
+import os
+
+import inspect
+import traceback
+from pprint import pprint
+
+
+# Setting this environment variable will enable the dump of the debugging
+# call stack
+DIRAC_DEBUG_DENCODE_CALLSTACK = bool(os.environ.get('DIRAC_DEBUG_DENCODE_CALLSTACK', False))
```

Comment:
This means any non-empty value for the environment variable will enable the printing, not just 'True'

Reply:
Correct. I am assuming only sensible persons that would use only True or False :D What do you recommend ? That I keep this behavior and change the doc, or enforce the env variable to be True ?

Reply:
Well exporting the environment variable to "False" would still enable it, because os.environ only contains strings as the values and any non-empty string is `True`. But I don't think the behaviour needs to be changed, just needs to be clear that exporting 0/False/Off still enables this feature.

Reply:
Silly me. When codding at 5am, one tends to forget that bool does not do a lexical cast
```
In [3]: bool('False')
Out[3]: True
```

:D 
Okay, fixing all that. Thanks for the review